validator_client/src/lib.rs

```diff
@@ -325,7 +325,7 @@ impl<E: EthSpec> ProductionValidatorClient<E> {
                     get_validator_block: slot_duration / HTTP_GET_VALIDATOR_BLOCK_TIMEOUT_QUOTIENT,
                 }
             } else {
-                Timeouts::set_all(slot_duration)
+                Timeouts::set_all(slot_duration.saturating_mul(config.long_timeouts_multiplier))
```

Comment:
This would still use the default timeouts if fallback node exists, is this the desired behaviour for this flag?

Reply:
This `else` branch is only used if either there is no fallback or if `--use-long-timeouts` is set.
If no fallback node exists and `--use-long-timeouts` is not set then it will still use the default multiplier of `1` which is the same as just being `slot_duration`. Since the value is not configurable unless `--use-long-timeouts` is set, I figured this was fine

Reply:
ah right, sorry I got confused. So it would never use the quick timeouts as long as `config.use_long_timeouts` is `true`, makes sense.